examples/python/rnnlm.py

```diff
@@ -98,7 +98,7 @@ def sample(self, first=1, nchars=0, stop=-1):
     VOCAB_SIZE = vocab.size()
 
     model = Model()
```

Comment:
@pmichel31415 Good point! I think we need to make a full sweep through the Python examples to make them consistent.